### PR TITLE
Added support for docker-machine 0.5.1

### LINKED ISSUE #148: (Installing and)Synching gets to "Testing if docker machine is running" and stops
First, thanks so much for putting this together. It is _precisely_ what I've been struggling with over the weekend.

Following the steps in the readme, I created a machine using `docker-machine` thus

`docker-machine create --driver virtualbox spaces`

```
Running pre-create checks...
Creating machine...
(spaces) OUT | Creating VirtualBox VM...
(spaces) OUT | Creating SSH key...
(spaces) OUT | Starting VirtualBox VM...
(spaces) OUT | Starting VM...
Waiting for machine to be running, this may take a few minutes...
Machine is running, waiting for SSH to be available...
Detecting operating system of created instance...
Detecting the provisioner...
Provisioning created instance...
Copying certs to the local machine directory...
Copying certs to the remote machine...
Setting Docker configuration on the remote daemon...
To see how to connect Docker to this machine, run: docker-machine env spaces
```

`eval "$(docker-machine env spaces)"`

`docker-osx-dev install`

```
2015-11-30 10:55:27 [INFO] Starting install of docker-osx-dev
2015-11-30 10:55:27 [INFO] Updating HomeBrew
Already up-to-date.
2015-11-30 10:55:34 [WARN] Cask is already installed by HomeBrew, skipping
2015-11-30 10:55:34 [WARN] Found command boot2docker, assuming Boot2Docker is already installed and skipping
2015-11-30 10:55:34 [WARN] Found command docker-compose, assuming Docker Compose is already installed and skipping
2015-11-30 10:55:34 [WARN] Found command docker-machine, assuming Docker Machine is already installed and skipping
2015-11-30 10:55:34 [WARN] Found command fswatch, assuming fswatch is already installed and skipping
2015-11-30 10:55:34 [WARN] Found command greadlink, assuming GNU core utilities is already installed and skipping
2015-11-30 10:55:34 [INFO] Testing if docker machine is running
```

`docker-osx-dev`

```
2015-11-30 10:55:48 [INFO] Using default sync paths: .
2015-11-30 10:55:48 [INFO] Complete list of paths to sync: /Users/<snip>/sites/ctlt/docker-test/data
2015-11-30 10:55:48 [INFO] Using default exclude paths: .git
2015-11-30 10:55:48 [INFO] Complete list of paths to exclude: .git
2015-11-30 10:55:48 [INFO] Complete list of paths to include: 
2015-11-30 10:55:48 [INFO] Starting docker-osx-dev file syncing
2015-11-30 10:55:48 [INFO] Testing if docker machine is running
```

Manually ran `docker-machine status spaces` and got `Running`

Then, in another tab

`docker run -d -p 80:80 -v /Users/<snip>/sites/ctlt/docker-test/data:/var/www richardtape/spacesnginx:latest nginx`

```
Unable to find image 'richardtape/spacesnginx:latest' locally
latest: Pulling from richardtape/spacesnginx
fa5be2806d4c: Pull complete 
0cd86ce0a197: Pull complete 
e9407f1d4b65: Pull complete 
c9853740aa05: Pull complete 
e9fa5d3a0d0e: Pull complete 
de3bdb6852ec: Pull complete 
73329e4a2aa1: Pull complete 
Digest: sha256:ed6b9cdd1c62ac4d2f9f24d8cb10eaa7b499d42609a19078147b1c2507bbe9fc
Status: Downloaded newer image for richardtape/spacesnginx:latest
e2c1eaa5f86af37497ad1996da526528b158e5a09d5efc1d91334b9ea55cd941
```

`docker ps -a`

```
CONTAINER ID        IMAGE                            COMMAND             CREATED             STATUS              PORTS                NAMES
e2c1eaa5f86a        richardtape/spacesnginx:latest   "nginx"             6 seconds ago       Up 6 seconds        0.0.0.0:80->80/tcp   reverent_goldberg
```

`docker-machine ip spaces`
`192.168.99.101`

Visit `192.168.99.101` in a browser and I see the index.html file in my shared data directory. However, if I update that file on the host (i.e. using Atom on my host mac to update `/Users/<snip>/sites/ctlt/docker-test/data/index.html`) the changes aren't reflected when I refresh the browser.

OSX: 10.10.5
Virtualbox: 5.0.10
Docker version 1.9.1, build a34a1d5
fswatch 1.7.0
readlink (GNU coreutils) 8.24

I'm probably doing something stupid...

### PR DESCRIPTION
**Pull request by richardtape**
The SSHUser and IPAddress have, in the json output from "docker inspect", moved from the Driver object to the Driver.Driver object. Fixes #148

### CODE REVIEW

File: src/docker-osx-dev [hunk 1]
```diff
@@ -421,6 +422,8 @@ function configure_docker_machine {
 # Initializes and starts up the docker-machine VM.
 #
 function init_docker_machine {
+  isrunning=is_docker_machine_running
```
Comment (brikis98) [inline src/docker-osx-dev]: Looks like some debug stuff got left in

**Comment by brikis98** [comment]
Thanks for the PR! One minor thing to clean up.

**Comment by richardtape** [comment]
Wow. I... But... Nothing. I got nothing. No excuses. Sorry! Fixed up.

**Comment by brikis98** [comment]
Haha, no worries, happens all the time. Thanks for the fix!

**Pull request merged**